backend/src/main/java/eventlistener/service/AppUserService.java

```diff
@@ -0,0 +1,24 @@
+package eventlistener.service;
+
+import eventlistener.security.model.AppUserDTO;
+import eventlistener.security.repo.AppUserRepo;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AppUserService {
+    AppUserRepo appUserRepo;
+
+    private final BCryptPasswordEncoder passwordEncrypter = new BCryptPasswordEncoder();
```

Comment:
I think it could be autowired since you created a bean for the password encoder further up